static/js/graphs_page.js

```diff
@@ -1990,7 +1992,374 @@ var graphPage = {
+            //as updates graph to reflect max k values allowed in subgraph
+            $("#slider1").slider({
+                value: 45, //initial 45
+                max: 200,
+                min: 1,
+                step: 1,
+                slide: function (event, ui) {
+                    $("#input_edge").val(ui.value);
+                    m_val = ui.value;
+                    if (m_val < 1) {
+                        m_val = 1;
+                        $(this).slider({
+                            value: 1
+                        });
+                    }
+                },
+                change: function (event, ui) {
+                    if (event.originalEvent) {
+                        graphPage.colaOption.edgeLengthChanged($("#input_node").val());
+                    }
+                }
+            });
+
+            $("#slider2").slider({
+                value: 5,
+                max: 50,
+                min: 1,
+                step: 1,
+                slide: function (event, ui) {
+                    $("#input_node").val(ui.value);
+                    m_val = ui.value;
+                    if (m_val < 1) {
+                        m_val = 1;
+                        $(this).slider({
+                            value: 1
+                        });
+                    }
+                },
+                change: function (event, ui) {
+                    if (event.originalEvent) {
+                        graphPage.colaOption.nodeSpaceChanged($("#input_node").val());
+                    }
+                }
+            });
+
+
+
+        },
+        nodeSpaceChanged: function(newNodeSpace) {
+            cola_node_space = newNodeSpace;
+            newGraphLayout = {
+                name: 'cola',
+                nodeSpacing: cola_node_space,
+                edgeLengthVal: cola_edge_length,
+                animate: true,
+                randomize: cola_randomize,
+                avoidOverlap: cola_avoid_overlap,
+                fit: cola_fit,
+                flow: cola_flow,
+                infinite: cola_infinite,
+                maxSimulationTime: 1500
+            }
+
+            graphPage.applyLayout(newGraphLayout);
+            console.log("after applying layout");
+
+        },
+        edgeLengthChanged: function(newEdgeLength) {
+            cola_edge_length = newEdgeLength;
+            newGraphLayout = {
+                name: 'cola',
+                nodeSpacing: cola_node_space,
+                edgeLengthVal: cola_edge_length,
+                animate: true,
+                randomize: cola_randomize,
+                avoidOverlap: cola_avoid_overlap,
+                fit: cola_fit,
+                flow:cola_flow,
+                infinite: cola_infinite,
+                maxSimulationTime: 1500
+            }
+
+            graphPage.applyLayout(newGraphLayout);
+            console.log("after applying layout");
+
+        },
+        overlapChanged : function (checked) {
```

Comment:
All event handlers should start with "on". Like onOverlapChange. Make these changes to other event handlers as well in colaOption module